apps/sms/views/conversation/js/conversation.js

```diff
@@ -2915,9 +2923,9 @@ var ConversationView = {
     }
 
     // Update thread timestamp. Will be removed in bug 958105.
-    var thread = Threads.active;
-    if (thread) {
-      thread.timestamp = Date.now();
+    if (this.currentThread) {
+      // TODO: Will need to replace with Conversation service method.
```

Comment:
we decided we don't want to change the timestamp anyway (https://bugzilla.mozilla.org/show_bug.cgi?id=958105)